src/main/java/frc/robot/subsystems/pivot/PivotIO.java

```diff
@@ -32,12 +37,32 @@ public default boolean isPivotWithinAcceptableError() {
     return false;
   }
 
+  /**
+   * Sets the output of the pivot
+   *
+   * @param output output value from -1.0 to 1.0
+   */
   public default void setPivotSpeed(double output) {}
 
+  /**
+   * Sets the voltage of the pivot motors
+   *
+   * @param volts the voltage
+   */
   public default void setVoltage(double volts) {}
 
+  /**
+   * sets the pivot angle(rotations) shooter
```

Comment:
capitalize docstring. I wouldn't include the unit in parenthisis as they are defined in the parameter.

Reply:
also it doesn't necessarily set it to the position in rotations. You have an arbitrary double as a parameter. So if you pass in degrees you would have to convert to the required units for controlling the system (which for the real pivot is rotations).

Reply:
Should we set it in rotations? I think the sim code converts from rotations to radians or whatever it uses currently